src/main/resources/spring/applicationContext-opensrp-web.xml

```diff
@@ -62,7 +62,7 @@
 
     <bean class="org.opensrp.web.config.SwaggerConfig"/>
 
-    <alias name="${multimedia_file_manager}" alias="multimedia_file_manager"/>
+    <alias name="${multimedia_file_manager}" alias="multimedia.file.manager"/>
```

Comment:
The `name` also has to be `multimedia.file.manager`.

Reply:
it has underscore `_` here https://github.com/OpenSRP/opensrp-server-configs/blob/master/assets/config/opensrp.properties#L68

Reply:
That is updated in [v2.1-configs](https://github.com/OpenSRP/opensrp-server-configs/blob/04df5883cf1cfc3b2a4616c1165c4a1167787f8f/assets/config/opensrp.properties#L76), and is what is expected by the [MultimediaService](https://github.com/OpenSRP/opensrp-server-core/blob/efa4bb808f3684e9dc5b37b97a35ca4aa79d50a6/src/main/java/org/opensrp/service/MultimediaService.java#L84).

Reply:
Then this should be specially committed to  master to configs, since its already committed on server-core

Reply:
Changes should always be committed to master

Reply:
I think we discussed this, and the consensus was that the `v2.1` branches would periodically be merged into master?